plugins/modules/kafka_connectors.py

```diff
@@ -92,14 +87,28 @@
 TIMEOUT_WAITING_FOR_TASK_STATUS = 30  # seconds
 
 
-def get_current_connectors(connect_url, timeout, username, password, client_cert, client_key):
+def get_headers(token, headers=None):
+    if headers is None and token == '':
+        return None
+    if token == '': # no bearer auth enabled hence no change in headers
+        return headers
+    if headers is None:
+        # Making default None is required as default value gets updated on subsequent function calls
+        headers = {}
+    bearer_header = {
+        'Authorization': 'Bearer {}'.format(token),
+        # Cant use f-string as this file runs with python2
```

Comment:
we don't support python2, what is the reference here?

Reply:
This file's  first line is #!/usr/bin/python. Which is making the use of python2. Thus when I tried using f-string it failed. I even tried to remove that line or change it to python3. But then it failed somewhere else. So we will have to spend some effort to make it fully python3 compliant

Reply:
I think you can create a ticket for this, but it's better to remove this comment from here. it will be confusing for customers

Reply:
sure